.travis.yml

```diff
@@ -14,15 +14,16 @@ env:
     - VERSION='1.8'
     - VERSION='1.9'
 script: lein with-profile dev,$VERSION test
-after_success:
-  - CLOVERAGE_VERSION=1.0.9 lein cloverage -e '^cljam.main' --codecov
-  - bash <(curl -s https://codecov.io/bash) -f target/coverage/codecov.json
 jobs:
   include:
+    - stage: coverage
+      jdk: oraclejdk8
+      script:
+        - CLOVERAGE_VERSION=1.0.9 lein cloverage -e '^cljam.main' --codecov
+        - bash <(curl -s https://codecov.io/bash) -f target/coverage/codecov.json
     - stage: deploy
       jdk: oraclejdk8
       script: skip
-      after_success: echo skip
```

Comment:
Is it because of an upgrade of Travis CI that this line's no longer necessary?
Seems working fine now, but I wonder why we introduced it.

Reply:
I added this line to avoid that coverage is executed on deploy stage because top-level `after_success` is also enabled on build stages. Now coverage stage doesn't use `after_success`, so this line isn't needed anymore.

Reply:
I get it now, thanks! 😄